File: 17_Command line arguments/CommandLineADD.c
#include<stdio.h>
#include<stdlib.h>
// stdlib contain atoi()

int main(int argc, char* argv[])
{	
	int no1 = 0,no2 = 0,ans = 0;
	
	printf("number of arguments : %d\n",argc);
	printf("file name : %s\n",argv[0]);
	
	if(argc == 3)
	{
		//atoi = ascii to integer
		
		no1 = atoi(argv[1]);
		no2 = atoi(argv[2]);
		
		ans = no1 + no2;
		printf("Addition is : %d",ans);
	}
}

File: 17_Command line arguments/CommandLineADD.java
import java.util.Scanner;

class CommandLineADD
{
	public static void main(String args[])
	{
		//parseInt = converts argument to a string
		
		int no1 = Integer.parseInt(args[0]);
		int no2 = Integer.parseInt(args[1]);
		
		int ans = no1 + no2;
		
		System.out.println("Addition is : " +ans);
	}
}